app/code/Magento/ImportExport/Ui/DataProvider/ExportFileDataProvider.php

```diff
@@ -51,11 +60,14 @@ public function __construct(
         \Magento\Framework\Api\FilterBuilder $filterBuilder,
         DriverInterface $file,
         Filesystem $filesystem,
+        File $fileIO,
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)